### DISCUSSION TITLE
10. TCP와 UDP가 사용되는 사례를 각각 설명해 주세요.

### DISCUSSION THREAD

**Discussion by eun-byeol** [jewelry-box/cs-interview#15] [General]
TCP와 UDP가 사용되는 사례를 각각 설명해 주세요.

**Comment by cutehumanS2** [comment]
TCP는 신뢰성과 데이터 순서 보장이 중요한 웹 브라우징(HTTP/HTTPS), 이메일(SMTP), 파일 전송(FTP)같은 애플리케이션에서 사용됩니다.

반면, UDP는 실시간 성능과 속도가 중요한 스트리밍 서비스, 온라인 게임, DNS 요청, DHCP, VoIP(인터넷 전화) 같은 애플리케이션에서 주로 사용됩니다.
